### PR TITLE
Fix more bugs in gl-object-get-calls.js

### PR DESCRIPTION
**Pull request by zhenyao**
PTAL

### CODE REVIEW

**Comment by toji** [comment]
LGTM!

File: sdk/tests/resources/matForWebGL2UniformShader.vert [hunk 1]
```diff
@@ -30,12 +30,14 @@ uniform mat3x4 mval3x4;
...
+
 void main()
 {
-    gl_Position = vec4(mval2x3 * vec2(1.0, 2.0), 0.0, 0.0)
-            + vec4(mval2x4 * vec2(1.0, 2.0), 0.0, 0.0)
-            + vec4(mval3x2 * vec3(1.0, 2.0, 3.0), 0.0)
-            + vec4(mval3x4 * vec3(1.0, 2.0, 3.0), 0.0)
-            + mval4x2 * vec4(1.0, 2.0, 3.0, 4.0)
-            + mval4x3 * vec4(1.0, 2.0, 3.0, 4.0);
+  position = vec4(mval2x3 * vec2(1.0, 2.0), 0.0); +
```
Comment (kenrussell) [inline sdk/tests/resources/matForWebGL2UniformShader.vert]: It looks like there's an incorrect semicolon on this line.

File: sdk/tests/resources/matForWebGL2UniformShader.vert [hunk 2]
```diff
@@ -30,12 +30,14 @@ uniform mat3x4 mval3x4;
...
+      mval2x4 * vec2(1.0, 2.0); +
```
Comment (kenrussell) [inline sdk/tests/resources/matForWebGL2UniformShader.vert]: Here too.

**Comment by kenrussell** [comment]
LGTM overall after the fact, but one minor issue which I think should be cleaned up in a follow-on pull request.

**Pull request merged**